unicorecmsecd/locale/eng_GB/LC_MESSAGES/unicorecmsskeleton.po

```diff
@@ -1,30 +0,0 @@
-#
```

Comment:
@miltontony I removed the translations but I don't know how to create the new ones, as `django-admin` or `./manage.py makemessages` won't work here. how can I make the `.po` files 😕